cmd/podmon/Makefile

```diff
@@ -12,35 +12,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 # Includes the following generated file to get semantic version information
-MAJOR=0
```

Comment:
Reason for this are done here is we do not have to build test folders etc when we build the image. I think we should keep these here, and remove from top Makefile

Reply:
Reason for this approach was because the go.mod and go.sum is present in parent directory and we can't cd into the podmon directory, otherwise the go.mod and go.sum file will not be copied into the first stage of dockerfile and there is a build failure, Previously we were build the binary in our local machine, so they were present and the binary was being created successfully. And in the second stage we are only copying the binary and no other files. This is the right approach